Packs/CommonScripts/Scripts/GenerateAsBuilt/GenerateAsBuilt.py

```diff
@@ -21,8 +19,6 @@
 DEMISTO_DEPENDENCIES_PATH = "/itemsdependencies"
```

Comment:
I think we can remove all these constants and add the URLs to the right commands

Reply:
what about this?

Reply:
@samuelFain Still don't see anything regarding this.
We don't need those constants

Reply:
Some of the constants are used in multiple places in the script, and also i think it makes the code more clear and understandable, should I delete the constants anyway?